examples/nlp/language_modeling/megatron_bart_pretraining.py

```diff
@@ -67,10 +71,10 @@ def main(cfg) -> None:
     if cfg.model.resume_from_checkpoint is not None:
         resume_from_checkpoint = cfg.model.resume_from_checkpoint
     else:
-        resume_from_checkpoint = trainer._checkpoint_connector.resume_from_checkpoint_fit_path
+        resume_from_checkpoint = trainer._checkpoint_connector._ckpt_path
     logging.info(f'Resuming training from checkpoint: {resume_from_checkpoint}')
 
-    trainer._checkpoint_connector = CheckpointConnector(trainer, resume_from_checkpoint=resume_from_checkpoint)
+    trainer._checkpoint_connector = _CheckpointConnector(trainer)
```

Comment:
Hi @titu1994 
Adrian from the Lightning team here. For this specific code block here, I believe reaching into the checkpoint connector should no longer be necessary in Lightning 2.0, because it is now possible to set the path like this: `trianer.ckpt_path = cfg.model.resume_from_checkpoint`

But if you have other uses of the CheckpointConnector that call into protected API, please let me know so I can study your use case 🎉

Reply:
@athitten should we follow this suggestion?

Reply:
Yes incorporated this.